wire/internal/wire/wire_test.go

```diff
@@ -70,6 +70,8 @@ func TestWire(t *testing.T) {
 	}
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
```

Comment:
This is racy unless you add `test := test` before `t.Run`

Reply:
Done, thanks for the catch!